parlai/tasks/light_dialog/build.py

```diff
@@ -28,9 +28,21 @@ def build(opt):
         url = 'https://dl.fbaipublicfiles.com/parlai/light/light-unseen-processed2.pkl'
         fname = 'light_unseen_data.pkl'
         build_data.download(url, dpath, fname)
+
+        # Download the environment dataset.
+        url = 'https://dl.fbaipublicfiles.com/parlai/light/light-environment.pkl'
```

Comment:
you can use http://parl.ai/downloads as an alias for https://dl.fbaipublicfiles.com/parlai/